plugins/scenario/hu.bme.mit.gamma.scenario.statechart.util/.gitignore

```diff
@@ -1 +1,2 @@
 /bin/
+/xtend-gen/
```

Comment:
It would be better, if there was a separate .gitignore in the xtend-gen folder. This way, the xtend-gen folder is not pushed to the repository, which can cause problems for the users cloning it and cannot find the xtend-gen folder in the project (even though, it is not the classpath).